Started GET "/" for 172.22.0.1 at 2022-04-19 08:42:33 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/favicon.ico" for 172.22.0.1 at 2022-04-19 08:42:33 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.24.0.1 at 2022-04-19 11:53:29 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/favicon.ico" for 172.24.0.1 at 2022-04-19 11:53:30 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.24.0.1 at 2022-04-19 12:24:26 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/favicon.ico" for 172.24.0.1 at 2022-04-19 12:24:26 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.24.0.1 at 2022-04-19 12:28:09 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/favicon.ico" for 172.24.0.1 at 2022-04-19 12:28:09 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.24.0.1 at 2022-04-19 12:29:39 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/favicon.ico" for 172.24.0.1 at 2022-04-19 12:29:40 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.24.0.1 at 2022-04-19 12:29:40 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/favicon.ico" for 172.24.0.1 at 2022-04-19 12:29:40 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.24.0.1 at 2022-04-19 12:29:41 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/favicon.ico" for 172.24.0.1 at 2022-04-19 12:29:41 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.24.0.1 at 2022-04-19 12:29:41 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/favicon.ico" for 172.24.0.1 at 2022-04-19 12:29:41 +0000
  
Mysql2::Error::ConnectionError (Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)):
  
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `connect'
mysql2 (0.5.3) lib/mysql2/client.rb:90:in `initialize'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `new'
activerecord (6.0.4.7) lib/active_record/connection_adapters/mysql2_adapter.rb:24:in `mysql2_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:887:in `new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:931:in `checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:910:in `try_to_checkout_new_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:871:in `acquire_connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:593:in `checkout'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:437:in `connection'
activerecord (6.0.4.7) lib/active_record/connection_adapters/abstract/connection_pool.rb:1125:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:221:in `retrieve_connection'
activerecord (6.0.4.7) lib/active_record/connection_handling.rb:189:in `connection'
activerecord (6.0.4.7) lib/active_record/migration.rb:562:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (4.2ms)[0m  [1m[35mCREATE DATABASE `app_development` DEFAULT CHARACTER SET `utf8mb4`[0m
  [1m[35m (0.7ms)[0m  [1m[35mCREATE DATABASE `app_test` DEFAULT CHARACTER SET `utf8mb4`[0m
Started GET "/" for 172.25.0.1 at 2022-04-20 05:05:46 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 26.2ms | Allocations: 290)
Completed 200 OK in 31ms (Views: 28.6ms | ActiveRecord: 0.0ms | Allocations: 1506)


Started GET "/favicon.ico" for 172.25.0.1 at 2022-04-20 05:05:46 +0000
  
ActionController::RoutingError (No route matches [GET] "/favicon.ico"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (0.6ms)[0m  [1m[35mCREATE DATABASE `app_development` DEFAULT CHARACTER SET `utf8mb4`[0m
  [1m[35m (0.4ms)[0m  [1m[35mCREATE DATABASE `app_test` DEFAULT CHARACTER SET `utf8mb4`[0m
Started GET "/" for 172.25.0.1 at 2022-04-20 05:06:37 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 7.3ms | Allocations: 179)
Completed 200 OK in 11ms (Views: 9.0ms | ActiveRecord: 0.0ms | Allocations: 729)


Started GET "/" for 172.25.0.1 at 2022-04-20 05:06:39 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 0.5ms | Allocations: 8)
Completed 200 OK in 2ms (Views: 1.1ms | ActiveRecord: 0.0ms | Allocations: 241)


Started GET "/" for 172.25.0.1 at 2022-04-20 05:19:10 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 0.6ms | Allocations: 8)
Completed 200 OK in 2ms (Views: 1.7ms | ActiveRecord: 0.0ms | Allocations: 240)


Started GET "/" for 172.25.0.1 at 2022-04-20 11:20:55 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 5.7ms | Allocations: 7)
Completed 200 OK in 24ms (Views: 15.0ms | ActiveRecord: 0.0ms | Allocations: 237)


Started GET "/" for 172.26.0.1 at 2022-04-20 11:23:19 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 14.2ms | Allocations: 290)
Completed 200 OK in 24ms (Views: 19.5ms | ActiveRecord: 0.0ms | Allocations: 1506)


Started GET "/home" for 172.27.0.1 at 2022-04-20 11:26:51 +0000
  
ActionController::RoutingError (No route matches [GET] "/home"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "//top" for 172.27.0.1 at 2022-04-20 11:27:17 +0000
  
ActionController::RoutingError (No route matches [GET] "/top"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "//hometop" for 172.27.0.1 at 2022-04-20 11:27:23 +0000
  
ActionController::RoutingError (No route matches [GET] "/hometop"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "//hometop" for 172.27.0.1 at 2022-04-20 11:27:34 +0000
  
ActionController::RoutingError (No route matches [GET] "/hometop"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.27.0.1 at 2022-04-20 11:27:37 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 8.5ms | Allocations: 183)
Completed 200 OK in 11ms (Views: 9.9ms | ActiveRecord: 0.0ms | Allocations: 855)


Started GET "/home/top" for 172.27.0.1 at 2022-04-20 11:30:41 +0000
Processing by HomeController#top as HTML
Completed 204 No Content in 2ms (ActiveRecord: 0.0ms | Allocations: 64)


Started GET "/home/top" for 172.27.0.1 at 2022-04-20 11:30:50 +0000
Processing by HomeController#top as HTML
Completed 204 No Content in 0ms (ActiveRecord: 0.0ms | Allocations: 27)


Started GET "/top" for 172.27.0.1 at 2022-04-20 11:30:58 +0000
  
ActionController::RoutingError (No route matches [GET] "/top"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.27.0.1 at 2022-04-20 11:31:11 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 8.8ms | Allocations: 178)
Completed 200 OK in 11ms (Views: 10.1ms | ActiveRecord: 0.0ms | Allocations: 712)


Started GET "/home/top" for 172.27.0.1 at 2022-04-20 11:36:36 +0000
Processing by HomeController#top as HTML
Completed 204 No Content in 0ms (ActiveRecord: 0.0ms | Allocations: 28)


Started GET "/home" for 172.27.0.1 at 2022-04-20 11:36:41 +0000
  
ActionController::RoutingError (No route matches [GET] "/home"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/home/top" for 172.27.0.1 at 2022-04-20 11:36:47 +0000
Processing by HomeController#top as HTML
Completed 204 No Content in 0ms (ActiveRecord: 0.0ms | Allocations: 26)


Started GET "/home/top" for 172.27.0.1 at 2022-04-20 11:37:26 +0000
Processing by HomeController#top as HTML
Completed 204 No Content in 0ms (ActiveRecord: 0.0ms | Allocations: 26)


Started GET "/" for 172.27.0.1 at 2022-04-20 11:37:30 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 0.4ms | Allocations: 8)
Completed 200 OK in 2ms (Views: 1.2ms | ActiveRecord: 0.0ms | Allocations: 242)


Started GET "/home/top" for 172.27.0.1 at 2022-04-20 12:06:42 +0000
Processing by HomeController#top as HTML
Completed 204 No Content in 0ms (ActiveRecord: 0.0ms | Allocations: 41)


Started GET "/" for 172.28.0.1 at 2022-04-21 08:48:54 +0000
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 43.9ms | Allocations: 290)
Completed 200 OK in 51ms (Views: 48.4ms | ActiveRecord: 0.0ms | Allocations: 1506)


Started GET "/favicon.ico" for 172.28.0.1 at 2022-04-21 08:48:54 +0000
  
ActionController::RoutingError (No route matches [GET] "/favicon.ico"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.29.0.1 at 2022-04-22 02:23:06 +0000
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.29.0.1 at 2022-04-22 02:23:43 +0000
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.29.0.1 at 2022-04-22 02:36:42 +0000
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.29.0.1 at 2022-04-22 02:36:42 +0000
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.29.0.1 at 2022-04-22 02:55:04 +0000
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 02:56:01 +0000
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 02:56:08 +0000
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 02:56:09 +0000
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (32.3ms)[0m  [1m[35mCREATE TABLE `schema_migrations` (`version` varchar(255) NOT NULL PRIMARY KEY)[0m
  [1m[35m (25.5ms)[0m  [1m[35mCREATE TABLE `ar_internal_metadata` (`key` varchar(255) NOT NULL PRIMARY KEY, `value` varchar(255), `created_at` datetime(6) NOT NULL, `updated_at` datetime(6) NOT NULL)[0m
  [1m[35m (0.7ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (0.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Migrating to CreateRooms (20220421091309)
  [1m[35m (0.9ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 03:03:16 +0000
  [1m[35m (0.3ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (1.0ms)[0m  [1m[35mCREATE DATABASE `app_development` DEFAULT CHARACTER SET `utf8mb4`[0m
  [1m[35m (0.5ms)[0m  [1m[35mCREATE DATABASE `app_test` DEFAULT CHARACTER SET `utf8mb4`[0m
  [1m[35m (0.2ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (0.3ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Migrating to CreateRooms (20220421091309)
  [1m[35m (0.5ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 03:08:55 +0000
  [1m[35m (0.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 04:20:24 +0000
  [1m[35m (7.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 04:40:10 +0000
  [1m[35m (2.9ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (0.8ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (0.6ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Migrating to CreateRooms (20220421091309)
  [1m[35m (1.0ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 04:41:04 +0000
  [1m[35m (0.8ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 04:41:06 +0000
  [1m[35m (0.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 04:41:06 +0000
  [1m[35m (0.8ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 04:41:06 +0000
  [1m[35m (0.7ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 04:41:07 +0000
  [1m[35m (0.4ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (0.4ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (0.6ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Migrating to CreateRooms (20220421091309)
  [1m[35m (0.7ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 04:41:57 +0000
  [1m[35m (0.7ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (0.9ms)[0m  [1m[35mCREATE DATABASE `app_development` DEFAULT CHARACTER SET `utf8mb4`[0m
  [1m[35m (0.4ms)[0m  [1m[35mCREATE DATABASE `app_test` DEFAULT CHARACTER SET `utf8mb4`[0m
  [1m[35m (0.2ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (0.8ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Migrating to CreateRooms (20220421091309)
  [1m[35m (0.8ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 04:42:47 +0000
  [1m[35m (0.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 04:54:06 +0000
  [1m[35m (1.6ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActiveRecord::PendingMigrationError (

Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=development

):
  
activerecord (6.0.4.7) lib/active_record/migration.rb:586:in `check_pending!'
activerecord (6.0.4.7) lib/active_record/migration.rb:564:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.0.4.7) lib/active_support/callbacks.rb:101:in `run_callbacks'
actionpack (6.0.4.7) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (0.4ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (0.4ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Migrating to CreateRooms (20220421091309)
  [1m[35m (32.4ms)[0m  [1m[35mCREATE TABLE `rooms` (`id` bigint NOT NULL AUTO_INCREMENT PRIMARY KEY, `name` varchar(255), `ruleId` varchar(255), `useId` int, `created_at` datetime(6) NOT NULL, `updated_at` datetime(6) NOT NULL)[0m
  [1m[35m (0.4ms)[0m  [1m[35mBEGIN[0m
  [1m[36mActiveRecord::SchemaMigration Create (3.2ms)[0m  [1m[32mINSERT INTO `schema_migrations` (`version`) VALUES ('20220421091309')[0m
  [1m[35m (4.2ms)[0m  [1m[35mCOMMIT[0m
Migrating to CreateRules (20220421094135)
  [1m[35m (18.4ms)[0m  [1m[35mCREATE TABLE `rules` (`id` bigint NOT NULL AUTO_INCREMENT PRIMARY KEY, `ruleId` int, `limitPeople` int, `deadline` varchar(255), `content` varchar(255), `created_at` datetime(6) NOT NULL, `updated_at` datetime(6) NOT NULL)[0m
  [1m[35m (0.4ms)[0m  [1m[35mBEGIN[0m
  [1m[36mActiveRecord::SchemaMigration Create (0.5ms)[0m  [1m[32mINSERT INTO `schema_migrations` (`version`) VALUES ('20220421094135')[0m
  [1m[35m (4.0ms)[0m  [1m[35mCOMMIT[0m
  [1m[36mActiveRecord::InternalMetadata Load (0.6ms)[0m  [1m[34mSELECT `ar_internal_metadata`.* FROM `ar_internal_metadata` WHERE `ar_internal_metadata`.`key` = 'environment' LIMIT 1[0m
  [1m[35m (0.3ms)[0m  [1m[35mBEGIN[0m
  [1m[36mActiveRecord::InternalMetadata Create (0.6ms)[0m  [1m[32mINSERT INTO `ar_internal_metadata` (`key`, `value`, `created_at`, `updated_at`) VALUES ('environment', 'development', '2022-04-22 05:00:58.454561', '2022-04-22 05:00:58.454561')[0m
  [1m[35m (2.8ms)[0m  [1m[35mCOMMIT[0m
  [1m[35m (0.3ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
  [1m[35m (0.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 05:01:03 +0000
  [1m[35m (0.6ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.0.4.7/lib/rails/templates/rails/welcome/index.html.erb (Duration: 12.0ms | Allocations: 187)
Completed 200 OK in 28ms (Views: 14.1ms | ActiveRecord: 0.0ms | Allocations: 1010)


  [1m[35m (0.5ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (0.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Migrating to DeviseCreateUsers (20220422052335)
  [1m[35m (21.1ms)[0m  [1m[35mCREATE TABLE `users` (`id` bigint NOT NULL AUTO_INCREMENT PRIMARY KEY, `email` varchar(255) DEFAULT '' NOT NULL, `encrypted_password` varchar(255) DEFAULT '' NOT NULL, `reset_password_token` varchar(255), `reset_password_sent_at` datetime, `remember_created_at` datetime, `created_at` datetime(6) NOT NULL, `updated_at` datetime(6) NOT NULL)[0m
  [1m[35m (15.9ms)[0m  [1m[35mCREATE UNIQUE INDEX `index_users_on_email`  ON `users` (`email`) [0m
  [1m[35m (9.0ms)[0m  [1m[35mCREATE UNIQUE INDEX `index_users_on_reset_password_token`  ON `users` (`reset_password_token`) [0m
  [1m[35m (0.3ms)[0m  [1m[35mBEGIN[0m
  [1m[36mActiveRecord::SchemaMigration Create (0.7ms)[0m  [1m[32mINSERT INTO `schema_migrations` (`version`) VALUES ('20220422052335')[0m
  [1m[35m (2.1ms)[0m  [1m[35mCOMMIT[0m
  [1m[36mActiveRecord::InternalMetadata Load (0.5ms)[0m  [1m[34mSELECT `ar_internal_metadata`.* FROM `ar_internal_metadata` WHERE `ar_internal_metadata`.`key` = 'environment' LIMIT 1[0m
  [1m[35m (0.4ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
  [1m[35m (0.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 05:26:41 +0000
  
NoMethodError (undefined method `devise_for' for #<ActionDispatch::Routing::Mapper:0x00007fe5f016b998>):
  
config/routes.rb:2:in `block in <main>'
config/routes.rb:1:in `<main>'
  [1m[35m (0.3ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (0.4ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  [1m[36mActiveRecord::InternalMetadata Load (0.4ms)[0m  [1m[34mSELECT `ar_internal_metadata`.* FROM `ar_internal_metadata` WHERE `ar_internal_metadata`.`key` = 'environment' LIMIT 1[0m
  [1m[35m (0.3ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
  [1m[35m (0.4ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 05:29:37 +0000
  
NoMethodError (undefined method `devise_for' for #<ActionDispatch::Routing::Mapper:0x00007fe5f005bf08>):
  
config/routes.rb:2:in `block in <main>'
config/routes.rb:1:in `<main>'
Started GET "/" for 172.30.0.1 at 2022-04-22 05:42:16 +0000
  [1m[35m (0.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  
ActionController::RoutingError (No route matches [GET] "/"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 05:42:47 +0000
  
NoMethodError (undefined method `devise_for' for #<ActionDispatch::Routing::Mapper:0x000055f5042eb400>):
  
config/routes.rb:2:in `block in <main>'
config/routes.rb:1:in `<main>'
  [1m[35m (0.4ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  [1m[35m (0.3ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  [1m[35m (0.3ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
  [1m[35m (0.2ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (0.3ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Migrating to DeviseCreateUsers (20220422052335)
  [1m[35m (11.8ms)[0m  [1m[35mDROP INDEX `index_users_on_reset_password_token` ON `users`[0m
  [1m[35m (9.6ms)[0m  [1m[35mDROP INDEX `index_users_on_email` ON `users`[0m
  [1m[35m (10.6ms)[0m  [1m[35mDROP TABLE `users`[0m
  [1m[36mActiveRecord::SchemaMigration Destroy (5.7ms)[0m  [1m[31mDELETE FROM `schema_migrations` WHERE `schema_migrations`.`version` = '20220422052335'[0m
  [1m[35m (0.6ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
  [1m[35m (0.5ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 05:53:19 +0000
  
NoMethodError (undefined method `devise_for' for #<ActionDispatch::Routing::Mapper:0x000055f503d03858>):
  
config/routes.rb:2:in `block in <main>'
config/routes.rb:1:in `<main>'
Started GET "/" for 172.30.0.1 at 2022-04-22 05:53:20 +0000
  
ActionController::RoutingError (No route matches [GET] "/"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 05:53:21 +0000
  
ActionController::RoutingError (No route matches [GET] "/"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 05:53:21 +0000
  
ActionController::RoutingError (No route matches [GET] "/"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 05:53:24 +0000
  
ActionController::RoutingError (No route matches [GET] "/"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 05:54:58 +0000
  
ActionController::RoutingError (No route matches [GET] "/"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
Started GET "/" for 172.30.0.1 at 2022-04-22 06:01:32 +0000
  
ActionController::RoutingError (No route matches [GET] "/"):
  
actionpack (6.0.4.7) lib/action_dispatch/middleware/debug_exceptions.rb:36:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.0.4.7) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `block in tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:28:in `tagged'
activesupport (6.0.4.7) lib/active_support/tagged_logging.rb:80:in `tagged'
railties (6.0.4.7) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/request_id.rb:27:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.0.4.7) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/static.rb:126:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.0.4.7) lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
railties (6.0.4.7) lib/rails/engine.rb:527:in `call'
puma (4.3.12) lib/puma/configuration.rb:228:in `call'
puma (4.3.12) lib/puma/server.rb:727:in `handle_request'
puma (4.3.12) lib/puma/server.rb:476:in `process_client'
puma (4.3.12) lib/puma/server.rb:332:in `block in run'
puma (4.3.12) lib/puma/thread_pool.rb:134:in `block in spawn_thread'
  [1m[35m (0.4ms)[0m  [1m[34mSELECT GET_LOCK('5277929974492897470', 0)[0m
  [1m[35m (1.2ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Migrating to DeviseCreateUsers (20220422062601)
  [1m[35m (22.5ms)[0m  [1m[35mCREATE TABLE `users` (`id` bigint NOT NULL AUTO_INCREMENT PRIMARY KEY, `email` varchar(255) DEFAULT '' NOT NULL, `encrypted_password` varchar(255) DEFAULT '' NOT NULL, `reset_password_token` varchar(255), `reset_password_sent_at` datetime, `remember_created_at` datetime, `created_at` datetime(6) NOT NULL, `updated_at` datetime(6) NOT NULL)[0m
  [1m[35m (13.2ms)[0m  [1m[35mCREATE UNIQUE INDEX `index_users_on_email`  ON `users` (`email`) [0m
  [1m[35m (9.5ms)[0m  [1m[35mCREATE UNIQUE INDEX `index_users_on_reset_password_token`  ON `users` (`reset_password_token`) [0m
  [1m[35m (0.5ms)[0m  [1m[35mBEGIN[0m
  [1m[36mActiveRecord::SchemaMigration Create (1.0ms)[0m  [1m[32mINSERT INTO `schema_migrations` (`version`) VALUES ('20220422062601')[0m
  [1m[35m (3.4ms)[0m  [1m[35mCOMMIT[0m
  [1m[36mActiveRecord::InternalMetadata Load (1.2ms)[0m  [1m[34mSELECT `ar_internal_metadata`.* FROM `ar_internal_metadata` WHERE `ar_internal_metadata`.`key` = 'environment' LIMIT 1[0m
  [1m[35m (0.4ms)[0m  [1m[34mSELECT RELEASE_LOCK('5277929974492897470')[0m
  [1m[35m (0.4ms)[0m  [1m[34mSELECT `schema_migrations`.`version` FROM `schema_migrations` ORDER BY `schema_migrations`.`version` ASC[0m
Started GET "/" for 172.30.0.1 at 2022-04-22 06:26:44 +0000
  
NoMethodError (undefined method `devise_for' for #<ActionDispatch::Routing::Mapper:0x00007fe5fc009b98>):
  
config/routes.rb:3:in `block in <main>'
config/routes.rb:1:in `<main>'
